automatic/kubernetes-cli/update.ps1

```diff
@@ -28,17 +28,22 @@ function global:au_GetLatest {
     $changelog_page = Invoke-WebRequest -Uri $changelogs -UseBasicParsing
     #Only the latest three minor versions are supported: https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md
     #However, there are prereleases before the version is released, so the latest four changelogs/minor versions need to be checked.
-    $minor_version_changelogs = $changelog_page.links | ? href -match "CHANGELOG-[\d\.]+\.md" | Select-Object -Expand href -First 4
-    
+    $minor_version_changelogs = $changelog_page.links `
+      | Where-Object href -match "CHANGELOG-(?<version>\d+\.\d+)\.md`$" `
+      | Select-Object -Expand href -Skip 1 -First 3
```

Comment:
This should work when there are three supported release versions, and one prerelease version, like if `1.22`, `1.23`, and `1.24` are released, and `1.25 has prerelease versions available.

But it would break once `1.25` is released if they do not have a `1.26` prerelease available.

Feel free to correct me if I am misunderstanding this.

Reply:
I'm not sure, but we will notice if/when this becomes a problem once 1.25/1.26 stable is (not) released.

BTW, there might be a better way for looking for the k8s versions, but I did not try to find it. I just made that part of the existing code more clear.

Reply:
This is now fixed. It will download the last three released version streams.